src/main/resources/liquibase/tenant/scripts/2020-11-19--18-00-create-request-metadata-table.xml

```diff
@@ -40,4 +40,18 @@
                              referencedColumnNames="request_id" onDelete="CASCADE" onUpdate="NO ACTION"/>
   </changeSet>
 
+  <changeSet id="2021-01-19--14-00-add-boolean-field-to-request-metadata" author="Illia Daliek">
+    <addColumn tableName="request_metadata_lb">
+      <column name="stream_ended" type="boolean" defaultValueBoolean="true">
```

Comment:
Is default value should be true for defaultValueBoolean?

Reply:
Yes